config/virtuoso/virtuoso-qa.ini

```diff
@@ -0,0 +1,249 @@
+;
+;  virtuoso.ini
+;
+;  Configuration file for the OpenLink Virtuoso VDBMS Server
+;
+;  To learn more about this product, or any other product in our
+;  portfolio, please check out our web site at:
+;
+;      http://virtuoso.openlinksw.com/
+;
+;  or contact us at:
+;
+;      general.information@openlinksw.com
+;
+;  If you have any technical questions, please contact our support
+;  staff at:
+;
+;      technical.support@openlinksw.com
+;
+;
+;  Database setup
+;
+[Database]
+DatabaseFile       = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.db
+ErrorLogFile       = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.log
+LockFile           = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.lck
+TransactionFile    = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.trx
+xa_persistent_file = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso.pxa
+ErrorLogLevel      = 7
+FileExtend         = 200
+MaxCheckpointRemap = 2000
+Striping           = 0
+TempStorage        = TempDatabase
+Syslog             = 1	; write to syslog in case of failures where writing to log is not possible
+
+[TempDatabase]
+DatabaseFile       = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso-temp.db
+TransactionFile    = /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso-temp.trx
+MaxCheckpointRemap = 2000
+Striping           = 0
+
+;
+;  Server parameters
+;
+[Parameters]
+ServerPort                 = 1111
+LiteMode                   = 0
+DisableUnixSocket          = 1
+DisableTcpSocket           = 0
+;SSLServerPort			= 2111
+;SSLCertificate			= cert.pem
+;SSLPrivateKey			= pk.pem
+;X509ClientVerify		= 0
+;X509ClientVerifyDepth		= 0
+;X509ClientVerifyCAFile		= ca.pem
+MaxClientConnections       = 10
+CheckpointInterval         = 60
+TraceOn                    = errors	; write traces
+O_DIRECT                   = 0
+CaseMode                   = 2
+MaxStaticCursorRows        = 5000
+CheckpointAuditTrail       = 0
+AllowOSCalls               = 0
+SchedulerInterval          = 10
+DirsAllowed                = ., /usr/local/virtuoso-opensource/share/virtuoso/vad
+ThreadCleanupInterval      = 10
+ThreadThreshold            = 10
+ResourcesCleanupInterval   = 10
+FreeTextBatchSize          = 100000
+SingleCPU                  = 0
+VADInstallDir              = /usr/local/virtuoso-opensource/share/virtuoso/vad/
+PrefixResultNames          = 0
+RdfFreeTextRulesSize       = 100
+IndexTreeMaps              = 256
+MaxMemPoolSize             = 200000000
+PrefixResultNames          = 0
+MacSpotlight               = 0
+MaxQueryMem                = 4G	; memory allocated to query processor ;;Niels said to 4g
+VectorSize                 = 1000	; initial parallel query vector (array of query operations) size
+MaxVectorSize              = 1000000	; query vector size threshold.
+AdjustVectorSize           = 0
```

Comment:
This being 0 here and 1 on production could cause differences in behaviour between them, I'm not familiar enough with these settings to know whether that's an issue.

Reply:
Same comment as above. I'm also not familiar enough, so I left these settings unchanged.